library/src/test/java/com/novoda/downloadmanager/lib/BatchStatusTests.java

```diff
@@ -9,16 +9,14 @@
 import android.os.Bundle;
 import android.support.annotation.NonNull;
 
-import org.junit.Test;
```

Comment:
actually it my fault - that'll teach me for not committing with Intellij (which organises imports on commit)